string node_name
string topic_name
string health_status
string message